.circleci/config.yml

```diff
@@ -114,7 +114,7 @@ jobs:
           command: curl -o wiremock.jar https://repo1.maven.org/maven2/com/github/tomakehurst/wiremock-standalone/2.27.1/wiremock-standalone-2.27.1.jar
       - run:
           name: Run wiremock
-          command: java -jar wiremock.jar --port 9091
+          command: java -jar wiremock.jar --port 9091 --jetty-acceptor-threads 1 --verbose
```

Comment:
This makes wiremock receive and process 1 request at a time (effectively single threaded), which prevents problems with the wiremock statemachine not working correctly (ie. the integration tests and the node app send requests to wiremock faster than it can process the first one including changing its state)